src/rpc/blockchain.h

```diff
@@ -27,6 +27,7 @@ struct NodeContext;
 } // namespace node
 
 static constexpr int NUM_GETBLOCKSTATS_PERCENTILES = 5;
+using coinsqlite_cb_t = std::function<std::string(const COutPoint&, const Coin&)>;
```

Comment:
Is this used anywhere?

Reply:
Ah, not anymore. Was going to have configurable fields like in the CSV PR, but decided against it otherwise things would get a little too complex here.

Will remove 👍

Reply:
I see this is now addressed in 84a6e28a8536fdc79796856e36be86a3bffe1848, thanks!